### DISCUSSION TITLE
Error: The default export is not a React Component in page: "/"

### DISCUSSION THREAD

**Discussion by alen899-my** [vercel/next.js#67943] [Help]
### Summary

![Screenshot 2024-07-19 160325](https://github.com/user-attachments/assets/afe06e2c-a836-40ed-bfe0-de66a08b7697)
![Screenshot 2024-07-19 160404](https://github.com/user-attachments/assets/7f81ef0b-78db-49f1-8e99-2bffe445d756)

### Additional information

```js
pls solve
```

**Comment by icyJoseph** [comment]
Did you resolve this? Maybe shutdown the server, delete .next, delete `node_modules` and reinstall them, and re run the server?
